py3status/modules/net_rate.py

```diff
@@ -11,6 +11,8 @@
     interfaces: comma separated list of interfaces to track
     interfaces_blacklist: comma separated list of interfaces to ignore
     precision: amount of numbers after dot
+    color: colorise output according to the value of 'up|down|total'
+    threshold_bad / threshold_degraded: thresholds for color change
```

Comment:
These should be in alphabetical order and threshold_bad / threshold_degraded should be documented individually.